src/services/api/search.js

```diff
@@ -2,12 +2,18 @@ import { apiBase } from 'services/config'
 import { SEARCH, SITES } from 'common/constants'
 
 const search = {
-  get: async ({ sitename, searchParams, pageParam, perPage = 48 }) =>
-    apiBase()
-      .get(
-        `${SITES}/${sitename}/${SEARCH}/?${searchParams}&page=${pageParam}&pageSize=${perPage}`,
-      )
-      .json(),
+  get: async ({ sitename, searchParams, pageParam, perPage = 48 }) => {
```

Comment:
To be consistent with our current convention of containing data manipulation and logic in the datahooks, please split this into two separate api services and move any needed conditional logic into the dataHooks that are consuming them. Cheers.